itou/www/apply/forms.py

```diff
@@ -392,7 +392,7 @@ class Meta:
     def clean_resume_link(self):
         resume_link = self.cleaned_data.get("resume_link")
         # ensure the CV has been uploaded via our S3 platform and is not a link to a 3rd party website
-        if resume_link and settings.S3_STORAGE_ENDPOINT_DOMAIN not in resume_link:
+        if resume_link and not resume_link.startswith(f"https://{settings.S3_STORAGE_ENDPOINT_DOMAIN}"):
```

Comment:
Il faudrait ajouter un `/` à la fin, pour éviter que la base du l’URL puissent être montée en tant que sous-domaine.
```suggestion
        if resume_link and not resume_link.startswith(f"https://{settings.S3_STORAGE_ENDPOINT_DOMAIN}/"):
```